Lib/test/support/logging_helper.py

```diff
@@ -0,0 +1,29 @@
+import logging.handlers
+
+class TestHandler(logging.handlers.BufferingHandler):
```

Comment:
It's only used in test_logging. Why not simply moving this class into test_logging?

We don't provide any backward compatibility warranty for test.support. It should not be used outside CPython code base.

Reply:
It is what I did originally. But then move it into a separate submodule on the request of @vsajip (`logging` maintainer, who added this class 10 years ago).